app/models/job.rb

```diff
@@ -1,4 +1,11 @@
 class Job < ApplicationRecord
   belongs_to :user, class_name: "User", foreign_key: :employer_id
   has_many :users, through: :bids
+  has_many :bids
+
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :budget, presence: true
+  validates :start_date, presence: true
+  validates :end_date, presence: true
```

Comment:
maybe validate that end date comes after start date?

Reply:
Ok sounds good! I will look into it

Reply:
seems good, thanks! will test together with my controllers

Reply:
love this. good job.